lib/tags/group-settings.tag

```diff
@@ -175,6 +176,7 @@
                                 }
                             } else {
                                 self.refs.title.classList.remove('graphjs-success');
+                                self.refs.title.removeAttribute('data-savesuccesstext');
```

Comment:
Please remove this change. self.refs.title refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above